deploy/kubernetes/deploy-driver.sh

```diff
@@ -15,6 +15,30 @@ set -o errexit
 
 readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
 readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
+#readonly BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
+
+function check_service_account()
+{
+	# Using bash magic to parse JSON for IAM
+	IAM_NAME=$(grep -Po '"client_email": *\K"[^"]*"' ${GCE_PD_SA_DIR}/cloud-sa.json | tr -d '"')
```

Comment:
alternatively:
`jq '.client_email' ${GCE_PD_SA_DIR}/cloud-sa.json | tr -d '"'`
I'm not sure if every system has `jq` installed but mine certainly does and I would prefer using a JSON parser for JSON over bash magic :)

Reply:
Two of my standard dev machines don't have jq by default, that's why I offered to convert to python (can also be useful for Windows K8S deployments).
I also don't have a problem converting to jq and adding a message if it is missing if you prefer